### PR TITLE
Completed container wrapper for commercial components

### PR DESCRIPTION
**Pull request by regiskuckaertz**
@kelvin-chappell

### CODE REVIEW

File: commercial/app/views/books/booksStandardV2.scala.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   omnitureId: Option[String],
   clickMacro: Option[String])(implicit request: RequestHeader)
 
-@containerWrapper {
+@containerWrapper(Seq(), "")(HtmlFormat.empty)(HtmlFormat.empty)(HtmlFormat.empty)(HtmlFormat.empty) {
```
Comment (kelvin-chappell) [inline commercial/app/views/books/booksStandardV2.scala.html]: You can use `Nil` for an empty `Seq` but I suppose this is going to be filled in eventually anyway.

File: commercial/app/views/containerWrapper.scala.html [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+  headerTitle: String,
+  headerClassNames: Option[Seq[String]] = None,
+  bodyClassNames: Option[Seq[String]] = None)(kicker: Html)(blurb: Html)(ctas: Html)(stamp: Html)(innards: Html)
 
-<p>before innards goes here</p>
-
-@innards
-
-<p>after innards goes here</p>
+<aside class="adverts @classNames map s"adverts--_"">
```
Comment (kelvin-chappell) [inline commercial/app/views/containerWrapper.scala.html]: Does this mapping work?

**Comment by kelvin-chappell** [comment]
:+1:

File: commercial/app/views/containerWrapper.scala.html (hunk 2 above)
Comment (regiskuckaertz) [inline commercial/app/views/containerWrapper.scala.html]: It doesn't complain at compile time... anything wrong?

Comment (kelvin-chappell) [inline commercial/app/views/containerWrapper.scala.html]: I think it will produce 
`adverts--_ adverts--_ adverts--_ ...`

Comment (kelvin-chappell) [inline commercial/app/views/containerWrapper.scala.html]: `s"adverts--$_"` or `{name => s"adverts--$name"}` should work

**Pull request merged**